admin/src/graphql/searchUsers.js

```diff
@@ -5,15 +5,15 @@ export const searchUsers = gql`
     $searchText: String!
     $currentPage: Int
     $pageSize: Int
-    $notActivated: Boolean
-    $isDeleted: Boolean
+    $emailChecked: Boolean
+    $deletedAt: Boolean
```

Comment:
As this is a nullable boolean that does not point directly to a db field is suggest to use `deleted` here

Reply:
I suggest `withDeleted` like the other call we already have and as described in the Issue.

Reply:
- renamed `notActivated` to `filterByActivated`
- renamed 'isDeleted` to `filterByDeleted`

Both reflects that it is nullable as well I guess.